addons/project/models/project.py

```diff
@@ -1177,11 +1178,8 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
         string='Customer', recursive=True, tracking=True,
         compute='_compute_partner_id', store=True, readonly=False,
         domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]")
+    is_display_partner = fields.Boolean("Display Partner", compute="_compute_is_display_partner")
```

Comment:
I would call it either `display_partner` or `should_display_partner`. It is more correct 'english wise'.